src/lib/Nav.svelte

```diff
@@ -1,11 +1,5 @@
 <script>
-	const menu = {
-		'home': ['work','live','play'],
-		'internships': ['internship opportunties','host a fellow'],
-		'apply': ['requirements','application'],
-		'about': ['edpa','retain alabama research','legal']
-	}
-
```

Comment:
you may have conflicts with some changes i just made and I accidentally committed directly to master instead of making PR cause im used to typing `git push` 🤣